libs/java/service/src/main/java/ca/bc/gov/tno/services/converters/ParseZonedDateTime.java

```diff
@@ -0,0 +1,61 @@
+package ca.bc.gov.tno.services.converters;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+
+/**
+ * Helper methods to parse ZonedDateTime values.
+ */
+public class ParseZonedDateTime {
```

Comment:
Had to add these to handle the various errors occurring during deserialization.  Java can't handle dates... it's embarrassing!